lib/com_err/lex.l

```diff
@@ -53,8 +53,6 @@ static int getstring(void);
 
 %}
 
-%option nounput
```

Comment:
Does this cause problems on systems where lex generates an `unput()`?

Reply:
Probably will just cause a warning?

Reply:
Could be also useful to check for flex in configure, and if we're using it, pass --nounput on the command line. Do we want Heimdal to be able to compile without unused function warnings?